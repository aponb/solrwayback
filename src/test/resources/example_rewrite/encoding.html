<html lang="en">
<head>
    <title>Some page with tricky encoding</title>
</head>
<body>
<script>
    someFunction([[{
        "playable_url":"https:\/\/example.com\/j0\u00253D&_nc_ohc=PNyMjb_o1",
    }]]);
</script>
</body>
</html>
