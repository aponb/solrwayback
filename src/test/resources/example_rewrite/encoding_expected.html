<html lang="en">
<head>
<title>Some page with tricky encoding</title>
</head>
<body>
<script>
someFunction([[{
"playable_url":"http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/j0%5cu00253d&_nc_ohc=pnymjb_o1",
}]]);
</script>
</body>
</html>